sdk/keyvault/ci.yml

```diff
@@ -65,3 +69,6 @@ stages:
           Value: '-DBUILD_TESTING=ON -DBUILD_SAMPLES=ON'
         - Name: Performance
           Value: '-DBUILD_TESTING=ON -DBUILD_SAMPLES=ON -DBUILD_PERFORMANCE_TESTS=ON'
+      CMakeSourceTestOptions:
```

Comment:
Remove this.
Use `CMakeTestOptions` to add the option

Reply:
the thing here is that i need a new set of things because the default images for the pipeline are contaminated by vcpkg's dark presence, i need to have a pure, untainted set of machines.

Reply:
> the pipeline are contaminated by vcpkg's dark presence

We want to know that even when there's VCPKG with dependencies, if I set the CMake option, we will get the source instead of using VCPKG.

The CMake option (when == ON ) should prevent CMake from looking the dependency from VCPKG and instead consume it from the source we fetch and build.